tests/data/commands.xml

```diff
@@ -0,0 +1,217 @@
+send: '\r\n'
+send: 'GET /20160918/instances/ocid1.instance.oc1.iad.abuwcljrcoaozqfqcqqcjqhqyf26w52a5d4dmwh4bpfmo457lkgbu5s47cia HTTP/1.1\r\nconnection: keep-alive\r\naccept-encoding: gzip, deflate\r\naccept: application/json\r\nuser-agent: Oracle-PythonSDK/2.1.2 (python 2.7.5; x86_64-Linux)\r\nopc-request-id: E8062F1661194E10BFB7DC1345F78257\r\nopc-client-info: Oracle-PythonSDK/2.1.2\r\ncontent-type: application/json\r\ndate: Thu, 24 Jan 2019 17:41:33 GMT\r\nhost: iaas.us-ashburn-1.oraclecloud.com\r\nauthorization: Signature algorithm="rsa-sha256",headers="date (request-target) host",keyId="ocid1.tenancy.oc1..aaaaaaaaojlj67jpdjvx7wszzrnwnooss3xxt66oldeyt2x35a6gthqdfr4q/ocid1.user.oc1..aaaaaaaahraqlb5i5c2wdmyjwpqoi5wujry25zjyjio2rlmne6hdvkwwfsba/72:58:bc:dc:8c:62:df:bf:1c:ee:49:f9:88:98:69:e7",signature="lK292UhXI+DU/TE1p4MZArFd0XH8vFiBv4zgYjSfkhHoTCFO+ieMZ+HzulpQlhVx/clt8+asUNZboTLC48HlKz9nieaU34nu1aFeA+2gqqApyv41sYOnjXWMNq5BVKW2M3LON7IB9kE4vvL1ly/Jb/bMSLewzd6a9VL98zQlml3ecLj9Baam/fQUexr7oEXUBb6PmdhdRzpZK6JAYTXCCkUrS/wwyUv+rrp0f8a8VZ86eNt5+u2tfuUSd2B7La66TwNZbDWL6VFJ3zIVwbL1ADjlYst8pDSJacTqXNbYHaNm3boDm8Fg0iM4OJNCFPcUJmRfjbcqRWtslZsAWaK+Bw==",version="1"\r\n\r\n'
+reply: 'HTTP/1.1 200 OK\r\n'
+header: Date: Thu, 24 Jan 2019 17:41:33 GMT
+header: Content-Type: application/json
+header: Content-Length: 1714
+header: Connection: keep-alive
+header: opc-request-id: E8062F1661194E10BFB7DC1345F78257/36FDD14A463BA26199EB99460CCD43B7/771F30B953ECE35A7E67CFC2DDAE3FB8
+header: ETag: c11f3e58f3052d6553b0b503c2c7b8c9acdab2646e60e595530017c6420ebdb5
+header: X-Content-Type-Options: nosniff
+send: 'GET /20160918/vnicAttachments/?instanceId=ocid1.instance.oc1.iad.abuwcljrcoaozqfqcqqcjqhqyf26w52a5d4dmwh4bpfmo457lkgbu5s47cia&amp;compartmentId=ocid1.compartment.oc1..aaaaaaaawzn525kzymvcso4rxdfkdgkoscbuqam42dwihe3kzm44iv7mjnta HTTP/1.1\r\nconnection: keep-alive\r\naccept-encoding: gzip, deflate\r\naccept: application/json, application/x-json-stream\r\nuser-agent: Oracle-PythonSDK/2.1.2 (python 2.7.5; x86_64-Linux)\r\nopc-request-id: 5FAA3E9581F9437FA81AE5A0150F77F1\r\nopc-client-info: Oracle-PythonSDK/2.1.2\r\ncontent-type: application/json\r\ndate: Thu, 24 Jan 2019 17:41:34 GMT\r\nhost: iaas.us-ashburn-1.oraclecloud.com\r\nauthorization: Signature algorithm="rsa-sha256",headers="date (request-target) host",keyId="ocid1.tenancy.oc1..aaaaaaaaojlj67jpdjvx7wszzrnwnooss3xxt66oldeyt2x35a6gthqdfr4q/ocid1.user.oc1..aaaaaaaahraqlb5i5c2wdmyjwpqoi5wujry25zjyjio2rlmne6hdvkwwfsba/72:58:bc:dc:8c:62:df:bf:1c:ee:49:f9:88:98:69:e7",signature="EzUXpK+iWu7Zee6dMFUOXRgrc6iDnO89H78myeZY4WQwL+gB8EjogD4FYVFsqvX/CCGeMJVxEUyAE7NtM9gnittfiJw2LA65qhWS94CJ9cLSyp1JaKcGyJ1iGBbJF9ZhsFYuEKjbTn5gfJvoAQaVJfaljuFk0yufq/Y0McONYER/xGwCQwhM0ZSAyRRBk2Cbx8JwX/PeM++KumJIRLagTA1LCHPXM9VXbwOP+Obp04+NG4pKS35WHAqAkXTjjpHYxMeFbd9Tft4CHToEZJRzxMUf0WHiW5WH3+bpwZbxWavn8PNckwCqrIPOo275M2994SRJ7eeWfqqSGBOPSYnJCQ==",version="1"\r\n\r\n'
```

Comment:
I am pretty sure we do not want to commit this to GitHub

Reply:
deleted